Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/FirebaseDatabase.kt

```diff
@@ -41,13 +42,13 @@ open class FirebaseDatabase {
             }
 
             row.room = roomNames!!.keys.first()
-            row.trackSortOrder = trackSortOrder;
-            row.speakerName = speakerNames!!.keys.joinToString(separator = ", ")
+            row.trackSortOrder = trackSortOrder
+            row.primarySpeakerName = primarySpeakerName
+            row.speakerNames = speakerNames!!.keys.toList()
             row.speakerCount = speakerNames!!.size
             row.talkDescription = description
             row.talkTitle = name
-            val firstSpeaker = speakerNames!!.keys.first()
-            row.photo = speakerNameToPhotoUrl!!.get(firstSpeaker)
+            row.photo = speakerNameToPhotoUrl!!.get(row.getFirstSpeaker())
```

Comment:
this can be primarySpeakerName instead of getFirstSpeaker() now

Reply:
ok